### PR TITLE
Play music via spotify after attaching nfc tag

### LINKED ISSUE #128: Trigger spotify api when attaching nfc-tag
Trigger the Spotify api when updating / patching the currentNFCTag from the backend.

### PR DESCRIPTION
**Pull request by anbraten**
Closes #128

### CODE REVIEW

File: packages/backend/src/services/authentication/authentication.hooks.ts [hunk 1]
```diff
@@ -32,10 +30,8 @@ export default {
           const spotifyAccessToken = data.access_token;
           const spotifyRefreshToken = data.refresh_token;
 
-          await app.service('users').patch(userId, { spotifyAccessToken, spotifyRefreshToken });
+          await ctx.app.service('users').patch(userId, { spotifyAccessToken, spotifyRefreshToken });
```
Comment (JanEberlein) [inline packages/backend/src/services/authentication/authentication.hooks.ts:33]: Is the parameter 'ctx' named by us? If yes, we should it rename for readability

**Review by tcoro — APPROVED** [review]

File: packages/backend/tsconfig.json [hunk 2]
```diff
@@ -1,14 +1,13 @@
...
+    "lib": ["ES2019"],
+    "module": "commonjs",
+    "target": "ES2019",
     "outDir": "dist",
     "rootDir": ".",
     "resolveJsonModule": true,
-    "baseUrl": ".",
-    "paths": {
-      "@/*": ["src/*"]
-    }
```
Comment (JanEberlein) [inline packages/backend/tsconfig.json:11]: For my own understanding: why was this done?

File: packages/backend/src/services/authentication/authentication.hooks.ts (hunk 1 above)
Comment (julianhahn) [reply packages/backend/src/services/authentication/authentication.hooks.ts:33]: It stands for 'context'. It's a pretty common syntax.

But we could rename it.

Comment (anbraten) [reply packages/backend/src/services/authentication/authentication.hooks.ts:33]: Good point. I will rename it to `context` as the official feathers [documentation](https://docs.feathersjs.com/guides/basics/hooks.html#quick-example) uses that as well.

File: packages/backend/tsconfig.json (hunk 2 above)
Comment (anbraten) [reply packages/backend/tsconfig.json:11]: I removed it for now as it is not properly supported / configured and was throwing some errors.

File: packages/backend/src/services/authentication/authentication.hooks.ts (hunk 1 above)
Comment (JanEberlein) [reply packages/backend/src/services/authentication/authentication.hooks.ts:33]: Thank you very much 😄

File: packages/backend/tsconfig.json (hunk 2 above)
Comment (JanEberlein) [reply packages/backend/tsconfig.json:11]: ok, thanks for the info 😃

**Review by JanEberlein — APPROVED** [review]
Thank you very much 😄

**Pull request merged**